roles/scenario1/templates/create_env.sh.j2

```diff
@@ -15,7 +15,12 @@ if ! $OC_COMMAND openstack image show {{ image_name }} > /dev/null 2> /dev/null;
     if [ ! -f {{ image_file_name }} ]; then
         curl -L {{ image_url }} > {{ image_file_name }}
     fi
-    $OC_COMMAND openstack image create {{ image_name }} --disk-format qcow2 --container-format bare --public < {{ image_file_name }}
+    {% if installer == 'podified' %}
+        {{ oc_bin }} -n {{ oc_namespace }} cp {{ image_file_name }} openstackclient:/tmp/{{ image_name }}.img
```

Comment:
This will work but I think a better way is to pass the file through stdin. I believe the command will pick it up from there if --file is not passed: https://github.com/openstack/python-openstackclient/blob/3f781cc3f993d5dbbbe0ba9a47e310aa446b9bd8/openstackclient/image/v1/image.py#L323

Reply:
Actually, I now notice that that's what it was doing before. I wonder if you have an error captured for why it failed?

Reply:
Is it that you miss `--stdin`?

Reply:
I think you should add `-i` to `oc`:

```
    -i, --stdin=false:
        Pass stdin to the container
```

Reply:
You're right. I changed that. Thx